File: verify2.py
from playwright.sync_api import sync_playwright
import time

def run():
    with sync_playwright() as p:
        browser = p.chromium.launch()

        # Mobile
        mobile_page = browser.new_page(viewport={"width": 375, "height": 812})
        mobile_page.goto("http://localhost:5173")
        time.sleep(2)
        # Click search icon
        mobile_page.click("nav button")
        time.sleep(1)
        mobile_page.screenshot(path="mobile_search.png")

        browser.close()

run()

File: verify2.mjs
import { chromium, devices } from 'playwright';

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext({
    ...devices['iPhone 12']
  });
  const page = await context.newPage();

  await page.goto('http://localhost:4173');
  await page.waitForLoadState('networkidle');
  await page.screenshot({ path: 'screenshot_home2.png' });

  await context.close();
  await browser.close();
  console.log('E2E test finished, screenshots generated.');
})();
